tests/onnxruntime/test_modeling.py

```diff
@@ -1402,22 +1656,46 @@ def test_compare_generation_to_io_binding(self, model_arch):
         gc.collect()
 
 
-class ORTModelForImageClassificationIntegrationTest(unittest.TestCase):
-    SUPPORTED_ARCHITECTURES_WITH_MODEL_ID = {
-        "vit": "hf-internal-testing/tiny-random-vit",
+class ORTModelForImageClassificationIntegrationTest(ORTModelTestMixin):
+    SUPPORTED_ARCHITECTURES = [
+        "beit",
+        "convnext",
+        "data2vec_vision",
+        "deit",
+        "levit",
+        "mobilenet_v1",
+        "mobilenet_v2",
+        "mobilevit",
+        # "perceiver",
+        "poolformer",
+        "resnet",
+        "segformer",
+        "swin",
+        "vit",
+    ]
```

Comment:
Is it not possible to infer that list automatically?

Reply:
@michaelbenayoun I would propose in a next PR to introduce in `optimum/onnxruntime` a dict of officially supported (aka tested) architectures for ORTModel. We would then use in here. Does this sound good to you? I can as well add it to this PR directly if you prefer.

We can't use the ones from `exporters/tasks.py` because some of the models supported in the ONNX export are not in ORTModel.

Reply:
Alright, perfect!